### DISCUSSION TITLE
Send pdf document akankah support caption?

### DISCUSSION THREAD

**Discussion by masbagusstwn** [kirimwa-id/forum#162] [Tanya Jawab]
Hali mas Rio..

Kemarin saya baru coba ngetes kirim pdf dengan melampirkan kata-kata atau bisa dikenal sebagai caption, dan berhasil.. pesan pdf tetap terkirim dan caption juga ikut terkirim seperti halnya mengirim pesan gambar.

Kira - kira akankan proses kirim document di API kirimwa nantinya akan support hal demikian?

Karena untuk saat ini ketika saya coba menambahkan caption, maka yg terjadi nama file nya malah menjadi caption yg diinput.

Terimakasih mas Rio, sukses selalu dengan API KirimWA nya yg sangat bermanfaat sekali :)

**Comment by rioastamal** [comment]
@masbagusstwn, setahu saya pengiriman PDF/document memang tidak mendukung caption. Sehingga parameter tersebut pada API KirimWA.id digunakan untuk set nama file-nya ketika mengirim tipe pdf/document.

**Reply by masbagusstwn** [reply]
Dear @rioastamal ,

Untuk pengguna android tidak ada kolom untuk mengetik caption document, karna setelah memilih document maka langsung terkirim.

Saya mencobanya melalui Whatsapp Desktop dan Whatsapp Web kira-kira seperti tangkapan gambar berikut.

Mungkin memang programming antara android dengan aplikasi desktop / web berbeda seperti halnya aplikasi perpesanan lainnya sehingga beda fitur diantara keduanya.

![image](https://user-images.githubusercontent.com/96308070/213911482-96416fed-ff70-4c33-ba30-f0091a0956bd.png)
![image](https://user-images.githubusercontent.com/96308070/213911507-4ce9edef-4093-4561-806d-750658cfedb9.png)
